specification/maps/resource-manager/Microsoft.Maps/stable/2021-02-01/maps-management.json

```diff
@@ -0,0 +1,1105 @@
+            }
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "Creators"
+        ],
+        "description": "Updates the Maps Creator resource. Only a subset of the parameters may be updated after creation, such as Tags.",
+        "operationId": "Creators_Update",
+        "x-ms-examples": {
+          "Update Creator Resource": {
+            "$ref": "./examples/UpdateMapsCreator.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AccountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/CreatorNameParameter"
+          },
+          {
+            "name": "CreatorUpdateParameters",
+            "in": "body",
+            "description": "The update parameters for Maps Creator.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CreatorUpdateParameters"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The Maps Creator was successfully updated.",
+            "schema": {
+              "$ref": "#/definitions/Creator"
+            }
+          },
+          "default": {
+            "description": "An unexpected error occurred.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Creators"
+        ],
+        "description": "Delete a Maps Creator resource.",
+        "operationId": "Creators_Delete",
+        "x-ms-examples": {
+          "Delete Creator Resource": {
+            "$ref": "./examples/DeleteMapsCreator.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AccountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/CreatorNameParameter"
+          }
+        ],
+        "responses": {
+          "202": {
```

Comment:
so there is no scenario in which a delete can be synchronously completed? adding this later will require a new api-version

Reply:
correct, that is the behavior.

Reply:
I think we will only support the async creator delete.

Reply:
I'm re-opening this @varunkumta, we found a case where we need to implement operationResults resource type for this to function e2e. In-light of that discovery, we're making a change here to only support the synchronous status codes + provisioningState described on the RPC addendum.